core/dynamic-tags/manager.php

```diff
@@ -245,20 +245,17 @@ public function get_tags() {
 			 *
 			 * Fires when Elementor registers dynamic tags.
 			 *
-			 * @since 2.0.9
+			 * @param Manager $this Dynamic tags manager.
 			 * @deprecated 3.5.0 Use `elementor/dynamic_tags/register` hook instead.
 			 *
-			 * @param Manager $this Dynamic tags manager.
+			 * @since 2.0.9
 			 */
-			// TODO: Uncomment when Pro uses the new hook.
-			//Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->do_deprecated_action(
-			//	'elementor/dynamic_tags/register_tags',
-			//	[ $this ],
-			//	'3.5.0',
-			//	'elementor/dynamic_tags/register'
-			//);
-
-			do_action( 'elementor/dynamic_tags/register_tags', $this );
+			Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->do_deprecated_action(
+				'elementor/dynamic_tags/register_tags',
+				[ $this ],
+				'3.5.0',
```

Comment:
@rami-elementor from which version it counts as deprecated?

Reply:
Deprecated in version 3.5. Legacy code removal in version 4.3. But we need to uncomment this code. 

@StyleShit you commented this code. What's you take on that?

Reply:
@rami-elementor should I leave it uncommented or should I comment it back?

Reply:
Uncomment. But again, we need to ask @StyleShit .

Reply:
It's OK, I commented it out because it broke the Editor when `ELEMENTOR_DEBUG=true` and the developer used an older Pro version.

Reply:
@StyleShit, So should I leave it uncommented or should I comment it back?

Reply:
Keep it uncommented